### PR TITLE
Xref the dream continues

### PR DESCRIPTION
**Pull request by jcayouette**
Only systems and users remaining for xrefs. @keichwa I have tested the build and it works fine.

### CODE REVIEW

**Comment by jcayouette** [comment]
Going to merge so I can package for RC1 no major changes other than links.  Bofh's link checker will verify them.

File: modules/reference/pages/admin/task-schedules.adoc [hunk 1]
```diff
@@ -37,41 +33,36 @@ Clears task engine (taskomatic) history data older than a specified number of da
...
-<<advanced.topics.cobbler>>.
+For more information refer to:
+xref:client-configuration:cobbler.adoc[Cobbler]
 
 
 menu:compare-configs-default:[]::
 Compares configuration files as stored in configuration channels with the files stored on all configuration-enabled servers.
 To review comparisons, click the menu:Main Menu[Systems] tab and click the system of interest.
 Go to menu:Configuration[Compare Files].
 For more information, refer to:
```
Comment (keichwa) [inline modules/reference/pages/admin/task-schedules.adoc]:
```suggestion
For more information, see:
```

File: modules/reference/pages/admin/task-schedules.adoc [hunk 2]
```diff
@@ -37,41 +33,36 @@ Clears task engine (taskomatic) history data older than a specified number of da
 
 menu:cobbler-sync-default:[]::
 Synchronizes distribution and profile data from {productname} to Cobbler.
-For more information on Cobbler, see
-// TODO update link for github
-<<advanced.topics.cobbler>>.
+For more information refer to:
```
Comment (keichwa) [inline modules/reference/pages/admin/task-schedules.adoc]:
```suggestion
For more information, see:
```

File: modules/reference/pages/admin/task-schedules.adoc (hunk 1 above)
Comment (keichwa) [inline modules/reference/pages/admin/task-schedules.adoc]:
```suggestion
For more information, see:
```

File: modules/reference/pages/audit/audit-openscap-overview.adoc [hunk 3]
```diff
@@ -104,7 +100,8 @@ This functionality will be adapted and enabled in a future release.
 . To perform a scan via the Web interface, log in to {productname} .
 . Click on menu:Systems[] and select the target system.
 . Click on menu:Audit[Schedule] .
-. Fill in the Schedule New XCCDF Scan form. See <<sm-audit-schedule>> for more information about the fields on this page.
+. Fill in the Schedule New XCCDF Scan form. For more information about the fields on this page, see:
```
Comment (keichwa) [inline modules/reference/pages/audit/audit-openscap-overview.adoc]:
```suggestion
. Fill in the Schedule New XCCDF Scan form.
For more information about the fields on this page, see:
```

File: modules/reference/pages/navigation/patch-alert-icons.adoc [hunk 4]
```diff
@@ -1,25 +1,16 @@
...
 If all patches are installed, there is just a [guimenu]``View All Patches`` link to open the menu:Main Menu[Patches] page.
-Refer to
-ifndef::env-github,backend-html5[]
-<<s3-sm-errata-details>>
-endif::[]
-ifdef::env-github,backend-html5[]
-<<reference-webui-patches.adoc#s3-sm-errata-details,Patch details>>
-endif::[]
-for more information.
+For more information refer to:
```
Comment (keichwa) [inline modules/reference/pages/navigation/patch-alert-icons.adoc]:
```suggestion
For more information, refer to:
```

File: modules/reference/pages/navigation/systems-selected.adoc [hunk 5]
```diff
@@ -1,17 +1,9 @@
...
-
 On the menu:Main Menu[Systems > Overview] page, if you mark the check box next to a system, the [guimenu]``system selected`` number on the right area of the top bar increases.
-This number keeps track of the systems you have selected for use in the System Set Manager (SSM); for more information, see to
-ifndef::env-github,backend-html5[]
-<<ref.webui.systems.ssm>>.
-endif::[]
-ifdef::env-github,backend-html5[]
-<<reference-webui-systems.adoc#ref.webui.systems.ssm, Systems Set Manager>>.
-endif::[]
+This number keeps track of the systems you have selected for use in the System Set Manager (SSM); for more information, refer to:
```
Comment (keichwa) [inline modules/reference/pages/navigation/systems-selected.adoc]:
```suggestion
This number keeps track of the systems you have selected for use in the System Set Manager (SSM).
For more information, refer to:
```

File: modules/reference/pages/software/channel-details.adoc [hunk 6]
```diff
@@ -60,13 +51,8 @@ image::channels_details_patches_channels.png[scaledwidth=80%]
...
 The list displays advisory types, names, summaries, and issue dates.
 Clicking an advisory name takes you to its [guimenu]``Patch Details`` page.
-for more information, see
-ifndef::env-github,backend-html5[]
-<<s3-sm-errata-details>>.
-endif::[]
-ifdef::env-github,backend-html5[]
-<<reference-webui-patches.adoc#s3-sm-errata-details, Patch Details>>.
-endif::[]
+for more information, see:
```
Comment (keichwa) [inline modules/reference/pages/software/channel-details.adoc]:
```suggestion
For more information, see:
```

File: modules/reference/pages/systems/activation-keys.adoc [hunk 7]
```diff
@@ -10,13 +10,8 @@ System-specific activation keys created through the [guimenu]``Reactivation`` su
...
 For more information about Activation Keys, see
-ifndef::env-github,backend-html5[]
-<<bp.key.managment>>.
-endif::[]
-ifdef::env-github,backend-html5[]
-<<bp_chap_activation_key_management.adoc#bp.key.managment, Activation Keys>>.
-endif::[]
-
+xref:client-configuration:creating-activation-keys.adoc[Creating Activation Keys]
+and: xref:client-configuration:client-and-activation-keys.adoc[Activation Keys]
```
Comment (keichwa) [inline modules/reference/pages/systems/activation-keys.adoc]:
```suggestion
and xref:client-configuration:client-and-activation-keys.adoc[Activation Keys]
```

**Pull request merged**